.cirrus.yml

```diff
@@ -28,12 +28,14 @@ env:
     #### Comment out fedora-35 for podman 4.x branches.
     ####
     FEDORA_NAME: "fedora-36"
+    FEDORA_AARCH64_NAME: "${FEDORA_NAME} {aarch64}"
     #PRIOR_FEDORA_NAME: "fedora-35"
     UBUNTU_NAME: "ubuntu-2204"
 
     # Image identifiers
     IMAGE_SUFFIX: "c6013173500215296"
-    FEDORA_AMI_ID: "ami-0f116746f31965e41"
+    FEDORA_AMI_ID: "ami-0f116746f31965e41"  # matches c6013173500215296, Cirrus task: 5241178026147840
```

Comment:
On your next update: please fix comment with ambiguous image IDs.  Also (ideally) all these images should come from the same build (same ID).  That lowers the amount of brain-power required when reviewing test results.  i.e. the `crun` version is the same in everything called Fedora.

Reply:
one of the commits from unmerged PRs had the `matches: foo` thing, while sometime back you mentioned adding the cirrus task ID next to the ami. Let me know which one you prefer.

RE: fedora, is the current one showing different crun versions?

Reply:
and ya, I haven't updated the build ids today

Reply:
Definitely reference either the `c/automation_images` build ID or the `$IMAGE_SUFFIX` (basically the same thing, but with a `c` in front).  I opened a Cirrus-CI RFE to help make this easier for humans, but until then we'll need to rely on comments :disappointed:

Reply:
In case it makes your life easier, [these ***just*** built, feel free to use them:](https://github.com/containers/automation_images/pull/157#issuecomment-1194567684)

|Name|ID|
|---|---|
|**build-push**|`build-push-c5495735033528320`|
|**fedora**|`fedora-c5495735033528320`|
|**fedora-aws**|`ami-0df5df528071f1052`|
|**fedora-netavark**|`fedora-netavark-c5495735033528320`|
|**fedora-netavark-aws-arm64**|`ami-00c26c58f6eaba850`|
|**fedora-podman-aws-arm64**|`ami-02ee8b3a782a78791`|
|**fedora-podman-py**|`fedora-podman-py-c5495735033528320`|
|**prior-fedora**|`prior-fedora-c5495735033528320`|
|**ubuntu**|`ubuntu-c5495735033528320`|

Reply:
@cevich thanks, as part of the RFE, if possible, would be nice to have this table constantly updated on some fixed bookmarkable link so we don't have to look at PRs or logs.